src/Spatial/Euclidean/Quaternion.cs

```diff
@@ -166,43 +136,46 @@ public bool IsRotation
         public EulerAngles ToEulerAngles()
         {
             return new EulerAngles(
-                Angle.FromRadians(Math.Atan2(2*(_w*_x + _y*_z), ((_w*_w) + (_z*_z) - (_x*_x) - (_y*_y)))),
-                Angle.FromRadians(Math.Asin(2*(_w*_y - _x*_z))),
-                Angle.FromRadians(Math.Atan2(2*(_w*_z + _x*_y), ((_w*_w) + (_x*_x) - (_y*_y) - (_z*_z)))));
+                Angle.FromRadians(Math.Atan2(2 * (_w * _x + _y * _z), ((_w * _w) + (_z * _z) - (_x * _x) - (_y * _y)))),
+                Angle.FromRadians(Math.Asin(2 * (_w * _y - _x * _z))),
+                Angle.FromRadians(Math.Atan2(2 * (_w * _z + _x * _y), ((_w * _w) + (_x * _x) - (_y * _y) - (_z * _z)))));
         }
 
         /// <summary>
         /// Returns a new Quaternion q with the Scalar part only.
         /// If you need a Double, use the Real-Field instead.
         /// </summary>
-        public Quaternion Scalar()
+        public Quaternion Scalar
         {
-            return new Quaternion(_w, 0, 0, 0);
+            get { return new Quaternion(_w, 0, 0, 0); }
         }
 
         /// <summary>
         /// Returns a new Quaternion q with the Vector part only.
         /// </summary>
-        public Quaternion Vector()
+        public Quaternion Vector
         {
-            return new Quaternion(0, _x, _y, _z);
+            get { return new Quaternion(0, _x, _y, _z); }
         }
 
         /// <summary>
         /// Returns a new normalized Quaternion u with the Vector part only, such that ||u|| = 1.
-        /// Q may then be represented as q = r*(cos(phi) + u * sin(phi)) = r*exp(phi*u) where r is the absolute and phi the argument of q.
+        /// Q may then be represented as q = r*(cos(phi) + u*sin(phi)) = r*exp(phi*u) where r is the absolute and phi the argument of q.
         /// </summary>
-        public Quaternion UnitVector()
+        public Quaternion NormalizedVector
         {
-            return ToUnitQuaternion(0, _x, _y, _z);
+            get { return ToUnitQuaternion(0, _x, _y, _z); }
         }
 
         /// <summary>
         /// Returns a new normalized Quaternion q with the direction of this quaternion.
         /// </summary>
-        public Quaternion Sign()
+        public Quaternion Normalized
```

Comment:
Method would be consistent [with other parts of the lib](https://github.com/mathnet/mathnet-spatial/blob/master/src/Spatial/Euclidean/Vector3D.cs#L293-L296)